test/instantiation/test_instantiate_all_pipelines.py

```diff
@@ -205,13 +205,14 @@ def test_instantiate_DWIConnectome(cmdopt):
 def test_instantiate_PETVolume(cmdopt):
 
     from clinica.pipelines.pet_volume.pet_volume_pipeline import PETVolume
+    from clinica.utils.pet import Tracer
```

Comment:
Might make sense to have it as a general import since you are using it in several functions. I don't think you'd get circular imports by doing so.

Reply:
True, although scoping imports at the function level help identifying which test depends on what. YMMV.